Input/input_sup6.yaml

```diff
@@ -1,7 +1,6 @@
 mirco_input:
   WarmStartingFlag: true
   RandomTopologyFlag: true
-  RandomSeedFlag: false
```

Comment:
I can totally see, that this parameter might be somehow redundant with `RandaomGeneratorSeed`. 

There might be cases, where one wants to keep the parameter `RandomGeneratorSeed` in an input file, but might want to switch it on/off, e.g. when switching from debugging to production runs. Can I pass a non-sensible number to `RandomGeneratorSeed`, e.g. `-1`, to deactivate the seed feature? Or would this require the boolean switch `RandomSeedFlag`?